utils/validateHrRegisterRequest.ts

```diff
@@ -0,0 +1,12 @@
+import * as yup from 'yup';
+
+export const registerHrSchema = yup.object().shape({
+  email: yup.string().email('Proszę podać poprawny adres email.').required('Email jest wymagany do rejestracji.'),
+  fullName: yup.string().required('Imię i nazwisko jest wymagane do rejestracji.'),
+  company: yup.string().required('Nazwę firmy jest wymagane do rejestracji.'),
+  maxReservedStudents: yup
+    .number()
+    .min(0, 'Minimalna liczba osób 0.')
```

Comment:
tu minimalna liczba osob wg specyfikacji to jeden, reszta danych poprawiona według ustaleń, dzięki